app/code/core/Mage/Catalog/Model/Resource/Url.php

```diff
@@ -27,6 +27,10 @@
  */
 class Mage_Catalog_Model_Resource_Url extends Mage_Core_Model_Resource_Db_Abstract
 {
+    const PRODUCT_ENTITY_TYPE_ID = 4;
+    const STATUS_ATTRIBUTE_ID = 96;
+    const DISABLED_PRODUCT_STATUS = 2;
```

Comment:
Please no hardcoded IDs ...

Use `Mage_Catalog_Model_Product_Status::STATUS_DISABLED` instead of `DISABLED_PRODUCT_STATUS `

Reply:
I removed const
https://github.com/OpenMage/magento-lts/pull/1405/commits/11755bceee77d494250ce79c3f89ff614762d206